packages/ui-components/src/chat/chat.tsx

```diff
@@ -217,6 +217,7 @@ function Chat(props: ChatProps): JSX.Element {
                         isEditable={didUserWriteMessage(message, props.activeUser)}
                     />
                 ))}
+                {props.loadingComponent}
```

Comment:
Is the expectation that the caller is supposed to set this prop to null while the chat is not loading? Might be worth making that clear in the prop docstring.

Reply:
Yes, that is what I was thinking, will update docstrings